src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personName } = useParams();
+
+  const changingEndOfLink = (currentPerson: Person) => {
+    return `${currentPerson.name.toLowerCase().replaceAll(' ', '-')}-${currentPerson.born}`;
+  };
+
+  const selectedPersonName
+  = people.find(
+    person => changingEndOfLink(person) === personName,
+  )?.name || null;
+
+  const personInTheList = (currentName: string | null): Person | null => {
```

Comment:
```suggestion
  const getPersonByName = (currentName: string | null): Person | null => {
```
it will be better name